azurelinuxagent/common/utils/networkutil.py

```diff
@@ -155,6 +185,18 @@ def __get_common_drop_command_params(wait, command, destination):
         cmd.extend(["conntrack", "--ctstate", "INVALID,NEW", "-j", "DROP"])
         return AddFirewallRules._add_wait(wait, cmd)
 
+    # This rule allows DNS TCP request to wireserver ip for non root users
+    @staticmethod
+    def get_iptables_accept_dns_tcp_request_command(wait, command, destination):
+        cmd = AddFirewallRules.__get_iptables_base_command(wait)
+        if command == AddFirewallRules.get_insert_command():
+            cmd.extend(['-t', 'security', '-I', 'OUTPUT', '-d', destination, '-p', 'tcp', '--destination-port', '53', '-j', 'ACCEPT'])
+        if command == AddFirewallRules.get_check_command():
+            cmd.extend(['-t', 'security', '-C', 'OUTPUT', '-d', destination, '-p', 'tcp', '--destination-port', '53', '-j', 'ACCEPT'])
+        if command == AddFirewallRules.get_delete_command():
```

Comment:
Though we are not using this if block so far , as a part of next iteration I have plans to move the delete command we generate in default.py in here , so this block will be used